docs/timeseries/initializing.rst

```diff
@@ -227,11 +250,15 @@ would for a |Table| object::
     2016-03-22T12:30:40.000     6.0
     2016-03-22T12:30:43.000     4.0
 
-Passing data during initialization
+.. EXAMPLE END
+
+Passing Data During Initialization
```

Comment:
I would maybe change the order here, and first have the data during initialization, and then the adding more data. But it wasn't me writing this docs the first place, so I may very well miss a point.

Reply:
It looks like this can be done without causing any confusion.